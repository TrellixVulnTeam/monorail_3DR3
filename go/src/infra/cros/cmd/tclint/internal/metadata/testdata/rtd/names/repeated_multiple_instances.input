{
    "remoteTestDrivers": [
        {
            "name": "remoteTestDrivers/tast"
        },
        {
            "name": "remoteTestDrivers/tast"
        },
        {
            "name": "remoteTestDrivers/tauto"
        },
        {
            "name": "remoteTestDrivers/tauto"
        }
    ]
}