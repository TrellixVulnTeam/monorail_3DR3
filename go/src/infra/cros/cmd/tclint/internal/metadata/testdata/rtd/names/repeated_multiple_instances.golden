[
"## Errors ##",
"RemoteTestDriver names must be unique, found repeated name(s): 'remoteTestDrivers/tast','remoteTestDrivers/tauto'"
]